doc/guides/maintaining-V8.md

```diff
@@ -414,6 +414,11 @@ This would require some tooling to:
 <sup>1</sup>Node.js 0.12 and older are intentionally omitted from this document
 as their support has ended.
 
+<sup>2</sup>At the moment the V8 tests still require Python 2. To run these
```

Comment:
```suggestion
<sup>2</sup>The V8 tests still require Python 2. To run these
```

Reply:
Yeah, that makes sense. 

Taking a look at the [documentation style guide](https://github.com/nodejs/node/blob/master/doc/guides/doc-style-guide.md), I think this follows the guideline, ["be direct."](https://docs.microsoft.com/en-us/style-guide/word-choice/use-simple-words-concise-sentences)